lib/typed_buffers.dart

```diff
@@ -11,7 +11,7 @@
 /// That means that using the [TypedDataView.buffer] getter is not guaranteed
 /// to return the same result each time it is used, and that the buffer may
 /// be larger than what the list is using.
-library dart.pkg.typed_data.typed_buffers;
```

Comment:
I *think* this should be `typed_data.typed_buffers`.

If it's not the name of the package, it should be prefixed with it.